prince-of-versions/src/main/java/co/infinum/princeofversions/loaders/NetworkLoader.java

```diff
@@ -21,7 +21,7 @@
     /**
      * Default request timeout in seconds.
      */
-    public static final int DEFAULT_NETWORK_TIMEOUT_SECONDS = 60;
+    public static final int DEFAULT_NETWORK_TIMEOUT_SECONDS = 10;
```

Comment:
Also, it would be better if we set this value in milliseconds. That way we do not need two constants.  You can use `TimeUnit.SECONDS.toMills(60);`